infra/gravity/opscenter.go

```diff
@@ -2,23 +2,34 @@ package gravity
 
 import (
 	"bytes"
+	"context"
 	"fmt"
 	"os/exec"
 	"strings"
 	"text/template"
 	"time"
 
 	"github.com/go-yaml/yaml"
+	"github.com/gravitational/robotest/lib/defaults"
+	"github.com/gravitational/robotest/lib/wait"
 	"github.com/gravitational/trace"
 	"github.com/sirupsen/logrus"
 )
 
 // getTeleClusterStatus will attempt to get the cluster status from an ops center
 // for now, it's just looking for the specific line of output from tele get clusters
 // but could parse the entire yaml structure if imported from gravity
-func getTeleClusterStatus(clusterName string) (string, error) {
-	out, err := exec.Command("tele", "get", "clusters", clusterName, "--format", "yaml").Output()
+func getTeleClusterStatus(ctx context.Context, clusterName string) (string, error) {
+	out, err := exec.CommandContext(ctx, "tele", "get", "clusters", clusterName, "--format", "yaml").Output()
 	if err != nil {
+		// tele get clusters will get a non zero exit status code if the cluster is not found, but we want to make sure
```

Comment:
```go
"tele get clusters" returns a non-zero exit code ...
```
I'd rather have this documented with a function instead of the comment:
```go
if isClusterNotFoundError(out) {
	return "", trace.NotFound("cluster not found")
}
```